.github/workflows/snapshot-release.yml

```
@@ -97,10 +104,25 @@ jobs:
       - name: Pull Request Notification
         uses: actions/github-script@v6
         env:
+          RELEASES: ${{ steps.changesets.outputs.result }}
           MESSAGE: ${{ steps.publish.outputs.result }}
         with:
           script: |
             console.log(process.env.MESSAGE);
+            let changeset = { releases: [] };
+            try {
+              changeset = JSON.parse(process.env.RELEASES);
+            } catch (e) {}
+            const message = 'The following packages have been released:'
```

```suggestion
            let message = 'The following packages have been released:'
```
`message` is a const, but it is being mutated later.

https://github.com/lilnasy/astro/actions/runs/6589260196/job/17903321737